File: app/src/main/java/me/smbduknow/transport/Route.java
package me.smbduknow.transport;

import java.io.Serializable;

public class Route implements Serializable, Comparable<Route> {

    public String id;
    public String label;
    public String type;
    public String typeLabel;

    @Override
    public int compareTo(Route another) {
        return id.compareTo(another.id);
    }
}

File: app/src/main/java/me/smbduknow/transport/model/Route.kt
package me.smbduknow.transport.model

data class Route(
        var id: String? = null,
        var label: String? = null,
        var type: String? = null,
        var typeLabel: String? = null
) : Comparable<Route> {

    override fun compareTo(other: Route): Int {
        return id!!.compareTo(other.id!!)
    }
}
